CHANGELOG.md

```diff
@@ -20,3 +20,6 @@ Performance improvements for JSON serialization.
 Features
 * Regular expression support for custom application event mapping `matcher.value`.
 * `matchAll` support for custom application event mapping matcher.
+
+## 1.4
+* Address Log4J Arbitrary Code Execution exploit
```

Comment:
You can also reference the upstream CVE with more details here - https://nvd.nist.gov/vuln/detail/CVE-2021-44228